apps/3d_rec_framework/include/pcl/apps/3d_rec_framework/pc_source/registered_views_source.h

```diff
@@ -289,14 +278,7 @@ class RegisteredViewsSource : public Source<PointInT> {
         m.id_ = strs[0];
       }
       else {
-        std::stringstream ss;
-        for (int i = 0; i < (static_cast<int>(strs.size()) - 1); i++) {
-          ss << strs[i];
-          if (i != (static_cast<int>(strs.size()) - 1))
-            ss << "/";
-        }
-
-        m.class_ = ss.str();
+        m.class_ = boost::algorithm::join(strs, '/');
```

Comment:
Lovely

hDoes it need to incldue any Headers?

Reply:
I could compile it with 3 compilers, so I the header seems to be already included.